tasks/configuration.yml

```diff
@@ -1,5 +1,14 @@
 # tasks file
 ---
+- name: configuration | fail
+  fail:
+    msg: "haproxy_global_nbproc is deprecated"
+  when:
+    - haproxy_version is version('2.3', '>=') and haproxy_version is version('2.5', '<')
+    - haproxy_global_nbproc > 1
+  tags:
+    - haproxy-configuration-warn-or-fail
+
```

Comment:
Is the following to working?

```
- name: configuration | warn or fail
  fail:
    msg: "haproxy_global_nbproc is deprecated"
  ignore_errors: haproxy_version is version('2.5', '<')
  when: 
    - haproxy_version is version('2.3', '>=')
    - haproxy_global_nbproc > 1
  tags: 
    - haproxy-configuration-warn-or-fail
```

Reply:
That seems more correct; `warn` for `2.3 >=` and `fail` for `2.5 >=`

Reply:
I got an error that `haproxy_version is version('2.5', '<')` doesn't return a bool.

Reply:
I agree that it would be more correct this way.

Reply:
Maybe like this?

```yml
ignore_errors: "{{ haproxy_version is version('2.5', '<') }} "
```

Reply:
That worked. TIL a new trick in Ansible, thank you for that.